src/Builders/Payload/SendEventPayloadBuilder.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Abrouter\Client\Builders\Payload;
+
+use Abrouter\Client\Entities\JsonPayload;
+use Abrouter\Client\DTO\SummarizeEventDTO;
+use Abrouter\Client\DTO\IncrementEventDTO;
+
+class SendEventPayloadBuilder
+{
+    /**
+     * @param IncrementEventDTO $incrementEventDTO
+     * @return JsonPayload
+     */
+    public function buildSendIncrementEventRequest(IncrementEventDTO $incrementEventDTO): JsonPayload
```

Comment:
You can do sending in the one method. The only difference is the value and type.

Reply:
It's probably because you need to create the interface for this kind of event and depend on it. Not on concrete implementation